### DISCUSSION TITLE
Swtich `feature` tests to `system` tests

### DISCUSSION THREAD

**Discussion by spaghetticode** [solidusio/solidus#4580] [New Features or Ideas]
After https://github.com/solidusio/solidus/pull/3552 was merged, it may be a good time to upgrade our integration `feature` tests to `system` tests. 

Hopefully, the test suite execution time may even improve, as with system tests we don't need to use the`database_cleaner` gem anymore. 

This post may help: https://medium.com/table-xi/a-quick-guide-to-rails-system-tests-in-rspec-b6e9e8a8b5f6

**Comment by gonzar11** [comment]
I will work on this!
